CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ endif()
 
 project(openvino_tokenizers)
 
+option(CPACK_INCLUDE_TOPLEVEL_DIRECTORY "Boolean toggle to include/exclude top level directory." OFF)
```

Comment:
I suppose `find_package(OpenVINODeveloperPackage)` can override this value, cannot it?
I would move it closer to cpack call

Also, it's not required to keep it as `option`.

Reply:
Why do we need this option at all? Are we going to build the extension differently?

Reply:
to have uniform outcome in all repos - all repos will not create top level directory

Reply:
Agree, let's do not create top level directory too, but without one more cmake option

Reply:
Removed. What's bad about `option`?

Reply:
what it's part of cmake-gui or CmakeCache.txt, while it's not our goal